### ISSUE TITLE
Astebreed (GOG) crashes before first level starts

### ISSUE THREAD

**Issue by CartoonFan** [Joshua-Ashton/d9vk#227]
Please describe your issue as accurately as possible. If you run into a problem with a binary release, make sure to test with latest `master` as well.

**Important:** When reporting an issue with a specific game or application, such as crashes or rendering issues, please include log files and a D3D11 Apitrace (see https://github.com/apitrace/apitrace) so that the issue can be reproduced. In order to create a trace, run `wine apitrace.exe trace -a dxgi YOURGAME.exe`. DO NOT use DXVK together with apitrace!

### Software information
Astebreed (GOG)

### System information
- GPU: Nvidia GeForce GTX 960
- Driver: 430.26.0
- Wine version: 4.12.1 TKG
- DXVK version: https://github.com/Joshua-Ashton/d9vk/commit/065aeac2ab2575943eed58da4189c99a06e4d233

### Apitrace file(s)
(Nvidia Graphics, Wine w/D9VK)
https://drive.google.com/file/d/1Sgpme_yEpc5UwOPFcXLtHA9GlMqof_KV/view?usp=sharing

(Nvidia Graphics, Wine w/wined3d)
https://drive.google.com/file/d/1pT9B4-OVbyudt35y9yYsKyv64OrH_u3x/view?usp=sharing

(AMD Graphics, Windows)
https://drive.google.com/file/d/1uleWEEP_VJbfg8dTI32Pm2mzjz4lqhWs/view?usp=sharing

### Log files
- d3d9.log:
[Astebreed_d3d9.log](https://github.com/Joshua-Ashton/d9vk/files/3390527/Astebreed_d3d9.log)

Hello. I wanted to let you know that Astebreed crashes somewhere around the first lines of dialogue on D9VK while displaying an "Information" popup that looks something like this:
![Astebreed_error_message_20190714-150743](https://user-images.githubusercontent.com/29664894/61190837-a7420c80-a657-11e9-9335-6e3127ea0f3f.png)

I assume the first part is some garbled Japanese, but the second part looks like some kind of size. Also, in previous versions of D9VK, there was a memory allocation error at pretty much the same place, so doitsujin/dxvk#1100 might be involved here too. Both the wined3d and the windows traces play back fine on the Nvidia machine.

EDIT: All three traces are now .tar.gz files, sorry for the inconvenience.

**Comment by Joshua-Ashton** [comment]
Unfortunately I don't have anything to open .tar.lz files. 🐸 Can you please use .tar.gz, .tar.xz or .7z in future?

**Comment by CartoonFan** [comment]
Ah, sorry about that. The .lz files are [lzip](https://www.nongnu.org/lzip/lzip.html) files, supposedly more future-proof than .xz, according to the author. .lz files don't seem to work well with file-roller, but .tar.lz files work fine here :/ I can definitely convert them over, though. Do the bz2 files work OK?

**Comment by Joshua-Ashton** [comment]
I have no idea I haven't tried those. Can we just keep to sane formats that I can open in 7-Zip please 🐸

**Comment by CartoonFan** [comment]
Ha ha, OK. I'll try to stay "sane".

**Comment by CartoonFan** [comment]
All three traces should be .tar.gz now. Hope this helps :smiley:

**Comment by CartoonFan** [comment]
FYI, it looks like 7-Zip supports BZ2, according to [7-zip.org](https://www.7-zip.org/).

**Comment by CartoonFan** [comment]
So, as of a recent commit, the game will sometimes skip the dialogue that causes the crash and go straight to gameplay. However, if I quit and try to start a new game again, it crashes at the above spot.

**Comment by CartoonFan** [comment]
Here's a log with the memory allocation error:
[Astebreed_d3d9_memoryallocation.log](https://github.com/Joshua-Ashton/d9vk/files/3420678/Astebreed_d3d9_memoryallocation.log)

There was also an error message about a dds file( probably a texture or something) when the crash occurred (before the first line of dialogue). Don't have a screenshot yet :(

**Comment by CartoonFan** [comment]
So, I found out today the skip to the second set of cutscenes is actually native behavior :flushed:  The first cutscene plays without crashes, so I feel safe closing this. The one thing I wanted to mention was that, due to recent changes, there were a lot of "DS is bigger than the render targets" from playing through just one stage. My log is now around 36 MB and has almost 86000 lines. Didn't see any visual issues, but I wanted to let you all know in case it's important.

Closing #227.
